src/utils/utils.ts

```diff
@@ -51,7 +52,8 @@ export const parseTriggerTitle = (title: string): parsedTriggerTitle => {
     return {
         title: match[0],
         instance: match[1],
```

Comment:
should we also check/defend agains `match` not having the correct number of items?

Reply:
It's not apparent from the diff, but I do a check for a valid regex match above the return using the values.

https://github.com/mattermost/mattermost-app-zendesk/pull/40/files/3c01f3453bead83e5e5d423b4a9f8eb502533fb1#diff-5acab41990585fa68ae01ff7b2e2915628d19237840c3061b31fcadc1c0f09b8R49-R51